edb/common/ast/base.py

```diff
@@ -234,8 +234,8 @@ def _check_type_real(type_, value, raise_error):
 
 
 class AST(object, metaclass=MetaAST):
-    __fields = []
-    __ast_frozen_fields__ = frozenset()
+    __fields = []  # type: List[str]
+    __ast_frozen_fields__ = frozenset()  # type: AbstractSet[str]
```

Comment:
Why type comments and not type annotations as usual?

Reply:
Oh, lol: because the struct mechanism interprets type annotations, which means that annotating these breaks everything!

(I had meant to write a comment explaining this and got distracted)

Reply:
Haha, yes, a comment is definitely needed here :)